letsencrypt/plugins/standalone.py

```diff
@@ -134,18 +134,12 @@ def supported_challenges_validator(data):
 
 
 class Authenticator(common.Plugin):
-    """Standalone Authenticator.
+    """Standalone Authenticator."""
 
-    This authenticator creates its own ephemeral TCP listener on the
-    necessary port in order to respond to incoming DVSNI and SimpleHTTP
-    challenges from the certificate authority. Therefore, it does not
-    rely on any existing server program.
-
-    """
     zope.interface.implements(interfaces.IAuthenticator)
     zope.interface.classProvides(interfaces.IPluginFactory)
 
-    description = "Standalone Authenticator"
+    description = "LE Will Automatically Test Your Domain"
```

Comment:
The problem is that the 'do x and stuff' (name) format comes from letsencrypt/plugins/disco.py
 49     def description_with_name(self):                                             
 50         """Description with name. Handy for UI."""                               
 51         return "{0} ({1})".format(self.description, self.name)
which would make it "Run a (standalone) weberver to probe that you control domains (....)
though I suppose we could change the display code to call 
 44     def description(self):                                                       
 45         """Description of the plugin."""                                         
 46         return self.plugin_cls.description      

thoughts?